docs/source/devel/index.rst

```diff
@@ -200,6 +200,38 @@ locks [#f1]_ . It will  will do:
    ?highlight=Task%20cookbook#ensuring-a-task-is-only-executed-one-at-a-time>`_
    . I avoid that two tasks write the same metadata, causing a race condition.
 
+
+RSTUF Worker Backend Services Development
+#########################################
+
+Storage
+=======
+
+The default RSTUF Worker source code is configured to use  `LocalStorage`.
+
+AWSS3
+-----
+
+1. Initiate the local development environment ``make run-dev``
+
+2. Initiate the localstack configuration ``make localstack-init``
+
+3. Update ``docker-compose.yml`` to use ``AWSS3``
+
+   * Set ``RSTUF_STORAGE_BACKEND`` as ``AWSS3``
+   * Uncomment the ``RSTUF_AWSS3_STORAGE*`` environment variables
+   * Comment the ``RSTUF_LOCAL_STORAGE*``
```

Comment:
I suggest you switch their order:
```suggestion
   * Comment the ``RSTUF_LOCAL_STORAGE*``
   * Uncomment the ``RSTUF_AWSS3_STORAGE*`` environment variables
```
as their order in `docker_compose_yml` is reversed:
![image](https://github.com/repository-service-tuf/repository-service-tuf-worker/assets/16246778/96b98dcc-091a-477e-9dd6-4aea03cb0524)

Reply:
It was replaced by the new docker-compose for AWS